packages/core/src/Modifiers.ts

```diff
@@ -1,6 +1,12 @@
 import { Modifier } from './Modifier';
 import { Constructor, isConstructor } from '../../utils/src/utils';
 
+/**
+ * When parsing dom elements, modifiers can be DOM inline nodes (a, b, strong).
+ *
+ * The order of modifiers are important because it impact the way they will
```

Comment:
"The order" -> "is"
"It" -> "impacts"

=> "The order of modifiers is important because it may impact their rendering." ?